model_api/cpp/models/src/image_model.cpp

```diff
@@ -87,6 +88,24 @@ ImageModel::ImageModel(std::shared_ptr<InferenceAdapter>& adapter, const ov::Any
     init_from_config(configuration, adapter->getModelConfig());
 }
 
+std::unique_ptr<ResultBase> ImageModel::inferImage(const ImageInputData& inputData) {
+    return ModelBase::infer(static_cast<const InputData&>(inputData));;
+}
+
+std::vector<std::unique_ptr<ResultBase>> ImageModel::inferBatchImage(const std::vector<ImageInputData>& inputImgs) {
+    std::vector<std::reference_wrapper<const InputData>> inputData;
+    inputData.reserve(inputImgs.size());
+    for (const auto& img : inputImgs) {
+        inputData.push_back(static_cast<const InputData&>(img));
+    }
+    ModelBase::inferBatch(std::vector<InputData>());
```

Comment:
Why do you infer empty data?

Reply:
That's a last-minute refactoring artifact, will delete